addons/account/views/account_move_views.xml

```diff
@@ -775,6 +775,9 @@
                                             'show_address': 1, 'default_is_company': True, 'show_vat': True}"
                                        options='{"always_reload": True, "no_quick_create": True}'
                                        attrs="{'invisible': [('move_type', 'not in', ('out_invoice', 'out_refund', 'in_invoice', 'in_refund', 'out_receipt', 'in_receipt'))]}"/>
+                                <field name="partner_shipping_id"
+                                        groups="account.group_delivery_invoice_address"
+                                        attrs="{'invisible': [('move_type', 'not in', ('out_invoice', 'out_refund', 'in_invoice', 'in_refund'))]}"/>
```

Comment:
Here it could conflict with the other change we want to do

Reply:
Yes, there is a conflict with this https://github.com/odoo/odoo/pull/80370

```suggestion
                                        attrs="{'invisible': [('move_type', 'not in', ('out_invoice', 'out_refund', 'out_receipt'))]}"/>